pom.xml

```
@@ -320,7 +341,7 @@
                         <dependency>
                             <groupId>com.puppycrawl.tools</groupId>
                             <artifactId>checkstyle</artifactId>
-                            <version>9.3</version>
+                            <version>10.1</version>
```

10.2 is the latest now:
```suggestion
                            <version>10.2</version>
```